sdcm/utils/adaptive_timeouts/load_info_store.py

```diff
@@ -249,6 +252,60 @@ def get(self, operation: str | None, timeout_occurred: bool | None = None):
         return [hit["_source"] for hit in res["hits"]["hits"]]
 
 
+class ArgusAdaptiveTimeoutStore(AdaptiveTimeoutStore):
+    """
+    Report adaptive timeout results to Argus.
+    """
+
+    def __init__(self):
+        self.test_config = TestConfig()
+        self.cycle_counters = defaultdict(count)
+
+    def send_adaptive_timeout_results_to_argus(self, results: dict):
+        argus_client = self.test_config.argus_client()
+        if not argus_client:
+            LOGGER.warning("Will not submit to argus - no client initialized")
+            return
+
+        class AdaptiveTimeoutResultsTable(GenericResultTable):
+            class Meta:
+                name = f"{results['operation']} - Timeout Statistics"
+                description = "measurement of specific operation timeouts (ex. decommission, adding nodes etc.)"
+                Columns = [
+                    ColumnMetadata(name="duration", unit="HH:MM:SS", type=ResultType.DURATION, higher_is_better=False),
+                    ColumnMetadata(name="timeout", unit="HH:MM:SS", type=ResultType.DURATION),
+                    ColumnMetadata(name="end_time", unit="", type=ResultType.TEXT),
+                ]
+        cycle = next(self.cycle_counters[results['operation']]) + 1
+        table = AdaptiveTimeoutResultsTable()
+        table.add_result(column="duration", row=f"#{cycle}", value=results['duration'], status=Status.UNSET)
```

Comment:
I think this one should have status error when we cross timeout, otherwise should be PASS. 'timeout' column should be always UNSET.

Reply:
Agree, red/errored duration cell is intuitively clear that it is bigger than expected.

Red/errored timeout cell is less clear.